DownFall/Tutorial/TutorialScene.swift

```diff
@@ -27,17 +27,29 @@ class TutorialScene: SKScene {
     //Generator
     private var generator: HapticGenerator?
     
+    //input to queue
+    private var inputQueue: InputType?
```

Comment:
This looks like the input queue, but really it is the input to queue after the game goes back to the playing state.  update the variable name and add a comment